easybuild/easyconfigs/y/yaml-cpp/yaml-cpp-0.8.0-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,30 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'CMakeMake'
+name = 'yaml-cpp'
+version = "0.8.0"
+
+homepage = "https://github.com/jbeder/yaml-cpp"
+description = """yaml-cpp is a YAML parser and emitter in C++ matching the YAML 1.2 spec"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/jbeder/%(name)s/archive/']
+sources = ['%(version)s.tar.gz']
+#patches = ['%(name)s-0.7.0_fix-cmake-export.patch']
+checksums = [
+    {'0.8.0.tar.gz': 'fbe74bbdcee21d656715688706da3c8becfd946d92cd44705cc6098bb23b3a16'},
+   # {'yaml-cpp-0.7.0_fix-cmake-export.patch': 'c562a575bbb161af478795e43bfd9073e3f874bea07134a321c1063cd7098c1c'},
```

Comment:
please remove patches if no longer relevant, otherwise update them as required

Reply:
done, not required anymore